### PR TITLE
Finished UI

### PR DESCRIPTION
**Pull request by zst-c**

### CODE REVIEW

File: mapping_project/lib/myhomepage.dart [hunk 1]
```diff
@@ -14,13 +15,27 @@ class MyHomePage extends StatefulWidget {
...
+    'assets/images/Music-1.svg',
+    'assets/images/Pepys-G.svg',
+    'assets/images/Pepys-1.svg',
+    'assets/images/Science-G.svg',
+    'assets/images/Science-1.svg',
+    'assets/images/Science-2.svg',
+    'assets/images/Science-3.svg',
   ];
-  var _moved = false;
+  //var _moved = false;
```
Comment (f11xter) [inline mapping_project/lib/myhomepage.dart]: why is this commented?

File: mapping_project/lib/myhomepage.dart [hunk 2]
```diff
@@ -30,10 +45,6 @@ class _MyHomePageState extends State<MyHomePage>
 
   List<String> suggestions = [
     "Saved",
```
Comment (f11xter) [inline mapping_project/lib/myhomepage.dart]: Is this the best way? Would it be better without an init value?

File: mapping_project/lib/myhomepage.dart [hunk 3]
```diff
@@ -112,175 +121,70 @@ class _MyHomePageState extends State<MyHomePage>
...
-                      ),
-                    ),
-                    Text(
-                      "\nExplore Your Timetable\n",
-                      style: TextStyle(color: Colors.black54),
-                    ),
-                  ],
-                ),
-              ),
+              //backdropColor: Theme.of(context).backgroundColor,
```
Comment (f11xter) [inline mapping_project/lib/myhomepage.dart]: why commented?

File: mapping_project/lib/saved_locs.dart [hunk 4]
```diff
@@ -1,159 +1,130 @@
...
-
-
-class _SavedLocsState extends State<SavedLocs> {//Actual savedlocs ofc
-
-
-
-final List<String> userLocs;
-
-_SavedLocsState(this.userLocs);
+  var userLocs = ["Room 1", "Room 2", "Room 3"];
```
Comment (f11xter) [inline mapping_project/lib/saved_locs.dart:19]: perhaps implement a getter from a file of the stored locations?

**Review by f11xter — CHANGES_REQUESTED** [review]
A couple of questions. I looked at all the changes made (not full additions of files) and everything looks fine. I'm assuming you've tested thoroughly.

File: mapping_project/lib/myhomepage.dart (hunk 1 above)
Comment (zst-c) [reply mapping_project/lib/myhomepage.dart]: This.... Shouldn't be commented - shall fix

File: mapping_project/lib/myhomepage.dart (hunk 2 above)
Comment (zst-c) [reply mapping_project/lib/myhomepage.dart]: This was for testing purposes, to show how search autocomplete shows when you type stuff in

File: mapping_project/lib/saved_locs.dart (hunk 4 above)
Comment (zst-c) [reply mapping_project/lib/saved_locs.dart:19]: Ah yes, missed that, will do later

Comment (zst-c) [reply mapping_project/lib/saved_locs.dart:19]: James is gonna do that actually - shall we merge and then he can edit it in?

**Review by f11xter — APPROVED** [review]

**Pull request merged**